doctor/serializers.py

```
@@ -0,0 +1,432 @@
+            'created_at', 'updated_at', 'is_active'
+        ]
+        read_only_fields = ['id', 'doctor_name', 'created_at', 'updated_at']
+    
+    def get_doctor_name(self, obj):
+        """نام پزشک"""
+        try:
+            return obj.doctor.doctor_profile.get_full_name()
+        except:
+            return obj.doctor.username
+    
+    def validate(self, data):
```

Ruff (0.12.2)

Missing return type annotation for public function `validate`

(ANN201)